backend/routers/admin_analytics.py

```
@@ -13,19 +13,51 @@
 
 @router.get("/analytics")
 async def analytics(request: Request, db=Depends(get_db)):
-    admin_user = AuthenticationHelper.verify_admin_access_strict(request)
-    return {
-        "users": await db.fetchval("SELECT COUNT(*) FROM users"),
-        "revenue": await db.fetchval("SELECT SUM(amount) FROM payments"),
-        "sessions": await db.fetchval("SELECT COUNT(*) FROM sessions"),
-    }
+    try:
+        admin_user = AuthenticationHelper.verify_admin_access_strict(request)
```

Ruff (0.12.2)

Local variable `admin_user` is assigned to but never used

Remove assignment to unused variable `admin_user`

(F841)